app/web/features/messages/requests/HostRequestUserSummarySection.tsx

```diff
@@ -0,0 +1,127 @@
+import { Skeleton, styled, Typography, useMediaQuery } from "@mui/material";
+import Avatar from "components/Avatar";
+import UserSummary from "components/UserSummary";
+import { useTranslation } from "i18n";
+import { MESSAGES } from "i18n/namespaces";
+import { LiteUser } from "proto/api_pb";
+import { HostRequest } from "proto/requests_pb";
+import { theme } from "theme";
+import { numNights } from "utils/date";
+import dayjs from "utils/dayjs";
+
+const StyledRequestedDatesWrapper = styled("div")(({ theme }) => ({
+  display: "flex",
+  "& > *": {
+    margin: 0,
+  },
+}));
+
+const StyledSmallUserSummary = styled("div")(({ theme }) => ({
+  display: "flex",
+  alignItems: "center",
+  borderBottom: `1px solid ${theme.palette.divider}`,
+  padding: theme.spacing(1, 2),
+}));
+
+const StyledLargeUserSummary = styled("div")(({ theme }) => ({
+  borderBottom: `1px solid ${theme.palette.divider}`,
+
+  [theme.breakpoints.down("md")]: {
+    borderBottom: `1px solid ${theme.palette.divider}`,
+    paddingBottom: theme.spacing(1),
+  },
+
+  [theme.breakpoints.up("sm")]: {
+    padding: theme.spacing(1),
+  },
+}));
+
+const StyledShortUserInfo = styled("div")(({ theme }) => ({
+  display: "flex",
+  flexDirection: "column",
+  marginLeft: theme.spacing(2),
+}));
+
+const HostRequestUserSummarySection = ({
+  hostRequest,
+  otherUser,
+}: {
+  hostRequest: HostRequest.AsObject | undefined;
+  otherUser: LiteUser.AsObject | undefined;
+}) => {
+  const { t } = useTranslation(MESSAGES);
+  const isMobile = useMediaQuery(theme.breakpoints.down("md"));
+
+  const smallUserSummarySection = (
+    <StyledSmallUserSummary>
+      {!otherUser ? (
+        <Skeleton variant="circular" sx={{ height: "2rem", width: "2rem" }} />
+      ) : (
+        <Avatar
+          style={{ height: "2rem", width: "2rem" }}
+          user={otherUser}
+          isProfileLink
+        />
+      )}
+      <StyledShortUserInfo>
+        <Typography component="p" variant="body2">
+          {!otherUser ? (
+            <Skeleton />
+          ) : (
+            `${
+              (otherUser?.name.length ?? 0) < 25
+                ? otherUser?.name
+                : otherUser?.name.substring(0, 25) + "..."
+            }, ${otherUser?.age}, ${otherUser?.city.split(",")[2]}` // get only country
```

Comment:
The user `city` isn't guaranteed to be of the format `City, State, Country`. I would say keep it as is, maybe truncate if it's too long.